packages/cadl-ranch-specs/http/models/property-types/main.cadl

```diff
@@ -172,4 +172,4 @@ interface DictionaryString extends ModelOperations<DictionaryStringProperty, "{'
 model NeverProperty is ModelTemplate<never>;
 @route("/never")
 @operationGroup
-interface Never extends ModelOperations<NeverProperty, "<dont include this property>"> {}
+interface Never extends ModelOperations<NeverProperty, "<don't include this property>"> {}
```

Comment:
fixed bc it failed cspell

Reply:
or did I merge without the ci passing...